app/_hub/kong-inc/ai-response-transformer/_metadata.yml

```diff
@@ -0,0 +1,22 @@
+name: AI Response Transformer
+search_aliases:
+  - ai
+  - llm
+  - artificial
+  - intelligence
+  - language
+  - model
+dbless_compatible: yes
+free: true
+enterprise: true
+paid: true
+konnect: true
+network_config_opts: All
+notes: --
+categories:
+  - ai
+publisher: Kong Inc.
+desc: Use a configured LLM service to introspect the upstream's HTTP(S) response before sending the response back to the client
```

Comment:
```suggestion
desc: Use a pre-configured LLM service to examine the upstream HTTP(S) prior to forwarding it to the client
```

Reply:
I don't think it's preconfigured, it's configured by the plugin.